setuptools/tests/test_distutils_adoption.py

```diff
@@ -53,7 +53,7 @@ def test_distutils_stdlib(venv):
     """
     Ensure stdlib distutils is used when appropriate.
     """
-    env = dict(SETUPTOOLS_USE_DISTUTILS='stdlib')
+    env = {**os.environ, **dict(SETUPTOOLS_USE_DISTUTILS='stdlib')}
```

Comment:
Sorry, just a curiosity question here: do the `sysconfig` installation schemes affect `virtualenv` installation schemes? (Previously my thought was that they only affect system-wide installations)

Reply:
If we are allowing other variables, I think we should be more specific here. For example it would not be nice to have things like `PYTHONHOME`, `PYTHONSTARTUP`, etc leaking...

Reply:
It shouldn't affect virtualenv, will recheck later. This part of the patch was written first, so maybe it is no longer necessary with the other part.

Reply:
Thank you @hroncok!

Reply:
As @hroncok said, I'm checking the patch without changes in `test_distutils_adoption` in Fedora's build environment and it still works fine. It's redundant here, I will remove it in an amended commit.